SparkPost/2web2ui
# FE-962: Suppressions page updates

### What Changed
 - Updated suppressions page and create page (under `/lists/suppressions` and `/lists/suppressions/create`
- Updated suppressions modal and 

### How To Test
 - Go to `/lists/suppressions` and `/lists/suppressions/create`
 - Check that search still works on page
 - Check that creation still works correctly
 - Check that pages match mocks
 - Check that details and delete modal matches mocks
